Uchu.Core/Packets/Server/GameMessages/VendorStatusUpdateMessage.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using RakDotNet;
+
+namespace Uchu.Core.Packets.Server.GameMessages
+{
+    public class VendorStatusUpdateMessage : ServerGameMessage
+    {
+        public override ushort GameMessageId => 0x1a1;
+
+        public bool UpdateOnly { get; set; }
+
+        public int[] LOTs { get; set; } = new int[0];
+        
+        public override void Serialize(BitStream stream)
+        {
+            Console.WriteLine($"Sending {GameMessageId} from {ObjectId}");
+            stream.WriteBit(UpdateOnly);
+            stream.WriteUInt32((uint) LOTs.Length);
+            for (var index = 0; index < LOTs.Length; index++)
```

Comment:
```suggestion
            foreach (var lot in LOTs)
```
just looks a bit nicer :)

Reply:
'index' is used in the GameMessage, using foreach would not be feasible. This should later the changed when we come to understand what SortPriority is.